README.md

```
@@ -373,148 +247,62 @@ python prototype.py --seed 42 --generations 3 --population 5 --duration 0.1
-- **Fitness metrics**: Candidate count, smoothness scores
-- **Timing breakdown**: Time spent on generation, filtering, and smoothness checks
-- **Rejection statistics**: Why candidates fail (modulus filter vs min hits)
-- **Example candidates**: Sample smooth numbers found
-
-#### Viewing Metrics
-
-Metrics are displayed in real-time during evolution:
+**Tracked Metrics**: Fitness (candidate count, smoothness scores), timing breakdown (generation/filtering/checking), rejection statistics, example candidates
 
+**Real-time Display**:
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)